run_snafu.py

```diff
@@ -35,7 +35,12 @@
 urllib3_log = logging.getLogger("urllib3")
 urllib3_log.setLevel(logging.CRITICAL)
 
-setup_loggers("snafu", logging.DEBUG)
+if os.getenv("snafu_debug") != None:
```

Comment:
Instead of this, can we follow the methodology that most python programs follow i.e. through simple argparse:
```
     parser.add_argument(
        "-v",
        "--verbose",
        dest="loglevel",
        help="set loglevel to INFO",
        action="store_const",
        const=logging.INFO)
    parser.add_argument(
        "-vv",
        "--very-verbose",
        dest="loglevel",
        help="set loglevel to DEBUG",
        action="store_const",
        const=logging.DEBUG)
```